pkgs/by-name/on/onionbalance/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  python313Packages,
+  fetchFromGitLab,
+}:
+
+python313Packages.buildPythonApplication rec {
+  pname = "onionbalance";
+  version = "0.2.4";
+
+  pyproject = true;
+  src = fetchFromGitLab {
+    domain = "gitlab.torproject.org";
+    owner = "tpo";
+    repo = "onion-services/onionbalance";
+    rev = version;
```

```suggestion
    tag = version;
```
To explicitly fetch a tag